src/mumble/Overlay_macx.mm

```diff
@@ -324,7 +321,7 @@ static bool authExec(AuthorizationRef ref, const char **argv) {
 	OSStatus err = noErr;
 	int pid = 0, status = 0;
 
-	err = AuthorizationExecuteWithPrivileges(ref, argv[0], kAuthorizationFlagDefaults, const_cast<char * const *>(&argv[1]), nullptr);
+	// Deprecated: AuthorizationExecuteWithPrivileges removed. Consider using SMJobBless or a helper tool for privileged operations.
```

Comment:
It seems like this effectively removes the entire functionality of `authExec` - in that case, we should return `false` (with a log message stating that this needs to be reimplemented.